tests/integration/test_client.py

```diff
@@ -272,6 +272,7 @@
 ):
     item = {"h": "h", "r": "r", "f": 1}
     await client.put_item(table, item)
-    await client.update_item(table, {"h": "h", "r": "r"}, F("f").set(2) & F("f").set(3))
-    item = await client.get_item(table, item)
-    assert item == {"h": "h", "r": "r", "f": 3}
+    with pytest.raises(errors.ValidationException):
```

Comment:
I wonder if `dynalite` also raises an exception in this case... 🤞

Reply:
it does. tests run against dynalite by default.